app/assets/javascripts/components/meetings_list.es6.jsx

```diff
@@ -0,0 +1,68 @@
+class MeetingsList extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.perPage = 12;
```

Comment:
Maybe this could be promoted to the component's `state` so it's dynamic (I can't see an use case now but maybe in the future).